src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -315,9 +311,6 @@ void AutomatedRssDownloader::updateRuleDefinitionBox(QListWidgetItem *selected)
             else
                 lMatch = tr("Last Match: Unknown");
             ui->lblLastMatch->setText(lMatch);
-            updateMustLineValidity();
-            updateMustNotLineValidity();
-            updateEpisodeFilterValidity();
```

Comment:
Oh wait - I think I see - it works because you're using `textChanged()`. But then it won't if you disconnect the signals as I suggested above.